recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
+{% set md5 = "c486211efa29b95771c74f5d8701ca4a" %}
```

Comment:
`sha256` from [pypi.org]( https://pypi.org/project/xgboost/0.6a2/#files ) is preferred.